build.sbt

```
@@ -0,0 +1,157 @@
+/*
+   Copyright 2013 Ilya Lakhin (Илья Александрович Лахин)
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+ThisBuild / organization := "name.lakhin.eliah.projects"
+ThisBuild / organizationName := "papacarlo"
+//ThisBuild / name := "Papa Carlo" // not used
+ThisBuild / version := "0.8.0-SNAPSHOT"
+ThisBuild / organizationHomepage := Some(url("http://lakhin.com/"))
+ThisBuild / scmInfo := Some(
+  ScmInfo(
+    url("https://github.com/your-account/your-project"),
+    "scm:git@github.com:Eliah-Lakhin/papa-carlo.git"
+  )
+)
+ThisBuild / developers := List(
+  Developer(
+    id = "Eliah-Lakhin",
+    name = "Ilya Lakhin",
+    email = "eliah.lakhin@gmail.com",
+    url = url("http://lakhin.com/")
+  )
+)
+ThisBuild / description := "Constructor of incremental parsers in Scala using PEG grammars"
+ThisBuild / licenses := List("Apache 2.0" -> new URL("http://www.apache.org/licenses/LICENSE-2.0.txt"))
+ThisBuild / startYear := Some(2013)
+ThisBuild / homepage := Some(url("http://lakhin.com/projects/papa-carlo/"))
+ThisBuild / pomIncludeRepository := { _ => false }
+ThisBuild / publishTo := {
+  val nexus = "https://oss.sonatype.org/"
+  if (isSnapshot.value)
+    Some("snapshots" at nexus + "content/repositories/snapshots")
+  else
+    Some("releases"  at nexus + "service/local/staging/deploy/maven2")
+}
+ThisBuild / publishMavenStyle := true
+
+// these versions must be static (no "latest.integration")
+ThisBuild / scalaVersion := "2.13.0"
+ThisBuild / crossScalaVersions := Seq("2.13.0")
+
+import sbt._
+import sbt.Keys._
+
+//addSbtPlugin("com.jsuereth" % "sbt-pgp" % "latest.integration")
+import com.jsuereth.sbtpgp._
+
+//addSbtPlugin("org.scala-js" % "sbt-scalajs" % "latest.integration")
+enablePlugins(ScalaJSPlugin)
+
+// "If you are using a Build.scala definition, import the following"
+import org.scalajs.sbtplugin.ScalaJSPlugin
+import org.scalajs.sbtplugin.ScalaJSPlugin.autoImport._
+
+
+
+  // used by JVM and JSDemo
+  lazy val baseSettings = Seq(
+    scalacOptions += "-unchecked",
+    //sourceDirectory := (sourceDirectory in PapaCarlo).value, // CIRCULAR
+  )
+
+
+
+  lazy val jsSettings =
+    Defaults.coreDefaultSettings ++
+    baseSettings ++
+    ScalaJSPlugin.projectSettings ++
```

You will need to remove those "default" settings. sbt plugins don't work like that anymore. The `.enablePlugins(ScalaJSPlugin)` is taking care of this instead.
```suggestion
    baseSettings ++
```